### DISCUSSION TITLE
Start RBM as service

### DISCUSSION THREAD

**Discussion by astronyu** [RainbowMiner/RainbowMiner#1692] [Q&A] [answered]
@RainbowMiner , how to start RBM is pause or standby mode? 
I sometimes use my gaming pc to mine and instead or exiting the script, how to make it goes standby and minimised with a click of button? That way I could start and stop the mining anytime. 
Not sure how the pause work.

**Comment by RainbowMiner** [comment, marked as answer]
Easy: set `"StartPaused": "1",` in config.txt

**Comment by astronyu** [comment]
Thank you.
